codegen/smithy-ruby-codegen-test/model/component-test/main.smithy

```diff
@@ -96,3 +97,62 @@ structure ClientError {
 @error("server")
 @retryable(throttling: true)
 structure ServerError {}
+
+@http(method: "POST", uri: "/default_kitchen_sink")
```

Comment:
I think a better home for this might be in component-test/defaults.smithy. Because we could put the default + non required test in there too.

Reply:
fixed